### PR TITLE
Add HASS Configurator to ecosystem

### PR DESCRIPTION
**Pull request by danielperna84**
**Description:**
Since my [HASS Configurator](https://github.com/danielperna84/hass-poc-configurator) has reached a stable state, I feel it's time to add it to the ecosystem section of the HASS site.
I don't think the amount of content justifies splitting the documentation up into several files. Hence the single-file-approach.
This has one drawback though, caused by the possible bug in Issue #2224 . It won't be listed in the topics right now. I assume as soon as the bug is fixed, this will be resolved automatically.

Any comments, suggested changes, rephrasing etc.? 🙂

### CODE REVIEW

File: source/_docs/ecosystem/hass-configurator.markdown [hunk 1]
```diff
@@ -102,7 +102,7 @@ Since the configurator script on its own is no service, you will have to take so
 2. If your system is using systemd (that's usually what you'll find on a Raspberry PI), there's a [template file](https://github.com/danielperna84/hass-poc-configurator/blob/master/hass-poc-configurator.systemd) you can use and then apply the same process to integrate it as mentioned in the [HASS documentation](https://home-assistant.io/docs/autostart/systemd/). If you use this method you have to set the `BASEPATH` variable according to your environment.
 3. If you have [supervisor](http://supervisord.org/) running on your system, [hass-poc-configurator.supervisor](https://github.com/danielperna84/hass-poc-configurator/blob/master/hass-poc-configurator.supervisor) would be an example configuration you could use to control the configurator.
 4. A tool called [tmux](https://tmux.github.io/), which should be pre-installed with recent AIO installers.
```
Comment (Landrash) [inline source/_docs/ecosystem/hass-configurator.markdown]: Tmx is also pre-installed with the HASSbian image.:)

**Comment by Landrash** [comment]
Looks good and can be merged. Up to you if you would like to mention the HASSbian image along side AiO installer under tmux. :)

**Comment by danielperna84** [comment]
Uhm, I actually though it was the AiO that had tmux. Could it be I'm mixing those two up and tmux actually is not part of the AiO and part of HASSbian instead?

**Comment by Landrash** [comment]
Checked the install script for AiO and I can't find any mention of tmux. I think it's only in later versions of HASSbian :)

**Comment by danielperna84** [comment]
Ok, took that out. Thanks for investigating. :)

**Comment by balloob** [comment]
If it is considered stable, you might want to consider renaming your repo to no longer have 'proof of concept' in the name. GitHub will auto redirect all old urls to the new name.

**Comment by danielperna84** [comment]
I didn't know that. Thanks for the hint. 🙂

**Comment by danielperna84** [comment]
Uhm, could it be that I should have used the current-branch as the target for this PR? My documentations for components/platforms always ended up online when the PR was against current and the new HASS release came out. This time though it seems to be missing.

**Comment by balloob** [comment]
Yes, you should have pointed it at `current` to go out right away, `next` to go out with next release. We cut release branch a couple of days before the actual release

**Comment by danielperna84** [comment]
Ah, ok. Then I'll just leave it this way since it's not urgent. Didn't notice the cut was made a few days before.

**Pull request merged**